component/vshn_appcat_services.jsonnet

```diff
@@ -179,6 +179,42 @@ local vshn_appcat_service(name, serviceParams) =
     },
   };
 
+
+  local scc =
+    {
+      allowHostDirVolumePlugin: true,
+      allowHostIPC: true,
+      allowHostNetwork: true,
+      allowHostPID: true,
+      allowHostPorts: true,
+      allowPrivilegeEscalation: false,
+      allowPrivilegedContainer: true,
+      allowedCapabilities: [
+        'MKNOD',
+        'CHOWN',
+        'SYS_CHROOT',
```

Comment:
Can you please explain why do we need root capability? We need to be very careful with this resource and make sure we give the least amount of privelegies.

Reply:
it's CHROOT - that's different thing https://man7.org/linux/man-pages/man2/chroot.2.html

Reply:
Can you please write here where these capabilities are required? Or we can have a short call.